src/pages/run-a-node.js

```diff
@@ -595,19 +595,22 @@ const RunANodePage = ({ data }) => {
           <Translation id="page-run-a-node-why-title" />
         </h2>
         <InfoGrid>
-          {whyRunANodeCards.map(({ image, title, preview, body, alt }, idx) => (
-            <ExpandableCard
-              contentPreview={<Translation id={preview} />}
-              title={<Translation id={title} />}
-              svg={image}
-              alt={<Translation id={alt} />}
-              key={idx}
-            >
-              {body.map((item, idx) => (
-                <p key={idx}>{<Translation id={item} />}</p>
-              ))}
-            </ExpandableCard>
-          ))}
+          {whyRunANodeCards.map(({ Image, title, preview, body, alt }, idx) => {
+            console.log({ Image })
+            return (
+              <ExpandableCard
+                contentPreview={<Translation id={preview} />}
+                title={<Translation id={title} />}
+                Svg={Image}
+                alt={<Translation id={alt} />}
```

Comment:
This need to be switched to `translateMessageId(alt)` since we are not rendering this as a component.